### PR TITLE
Refactored the Navbar with an avatar and a full bar.

### PR DESCRIPTION
**Pull request by Skorpios604**
This is closer to match the overall sought after design

### CODE REVIEW

File: Client/src/Components/NavBar/index.jsx [hunk 1]
```diff
@@ -1,48 +1,138 @@
...
-
-    return (
-        <div>
-            <Button
-                id="basic-button"
-                aria-controls={open ? 'basic-menu' : undefined}
-                aria-haspopup="true"
-                aria-expanded={open ? 'true' : undefined}
-                onClick={handleClick}
+const settings = ['Profile', 'Team', 'Invite Colleagues', 'Logout'];
```
Comment (popnfresh234) [inline Client/src/Components/NavBar/index.jsx:15]: Would these be passed in as props?  Not sure how often this component will be used, but if it is going to be used more than once I imagine they would be :thinking:

File: Client/src/Components/NavBar/index.jsx [hunk 2]
```diff
@@ -1,48 +1,138 @@
...
             >
-                <MenuItem onClick={handleClose}>Profile</MenuItem>
-                <MenuItem onClick={handleClose}>Team</MenuItem>
-                <MenuItem onClick={handleClose}>Invite Colleagues</MenuItem>
-                <MenuItem onClick={handleClose}>Log Out</MenuItem>
+              {settings.map((setting) => (
+                <MenuItem key={setting} onClick={handleCloseUserMenu}>
+                  <Typography textAlign="center">{setting}</Typography>
+                </MenuItem>
+              ))}
```
Comment (popnfresh234) [inline Client/src/Components/NavBar/index.jsx:129]: Nice, this is a good refactor

**Review by popnfresh234 — APPROVED** [review]
Nice refactor!

I'm not sure whether it makes sense to pass the items for the menu in as Props, but if the menu items are going to be defined in a higher level component then it probably does.

I'm sure that issue will make itself known if and when the time is ripe :laughing:

**Comment by Skorpios604** [comment]
Going to be passing in as props for now.

**Pull request merged**